content/articles/tera-template-engine.md

```diff
@@ -0,0 +1,136 @@
+Title: Introducing Tera, a template engine in Rust
+Date: 2016-04-14
+Short_summary: Talking about a template engine in Rust called Tera, inspired by Jinja2 and Django
+Category: Dev
+Authors: Vincent
+
+
+*Reading time: ~15 minutes.*
+
+Back in October 2015, I tried [Rust for web services](https://blog.wearewizards.io/trying-rust-for-web-services) and found the ecosystem lacking at the time. That's why I've been working on porting some of the tools we use in [Proppy](https://proppy.io/) to Rust: [jwt](https://crates.io/crates/jsonwebtoken), [bcrypt](https://crates.io/crates/bcrypt) (granted that Argon2 seems superior) and a [migration tool](https://crates.io/crates/dbmigrate). While I mostly do SPAs these days and don't write many templates in the backend, I still need one for some occasions. Being a Python, I like [Jinja2](http://jinja.pocoo.org/docs/dev/) and [Django templates](https://docs.djangoproject.com/en/1.9/topics/templates/#the-django-template-language). Here's how I attempted to port them to Rust and the result is [Tera](https://github.com/Keats/tera/).
```

Comment:
"Being a Python" sounds funny :snake:  :)

You can keep buy maybe you meant sth like "When using Python